### DISCUSSION TITLE
Internal Server Error

### DISCUSSION THREAD

**Discussion by harshh1802** [deta/cloud-docs#545] [Help]
Hello Everyone,
I have created basic FastApi app for blogs.
It is running fine on local but when I deployed it on deta response is "Internal Server Error" and Log is "No Errors". 
How to debug this?

It is fetching data but it is giving error when I try to add new data.

**Comment by malwaremanu** [comment]
Hi Harsh
can you share your code / repo ?

**Comment by harshh1802** [comment]
Hey,
Sorry I missed your email.

I checked documentation.
There was mentioned like it didn't support database writing.

Thanks
